torchchat/model.py

```diff
@@ -741,10 +770,9 @@ def apply_rotary_emb(x: Tensor, freqs_cis: Tensor) -> Tensor:
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 try:
-    from executorch.extension.pybindings import portable_lib as exec_lib
-    
     # ET changed the way it's loading the custom ops so it's not included in portable_lib but has to be loaded separately.
     from executorch.examples.models.llama2.custom_ops import sdpa_with_kv_cache  # no-qa
+    from executorch.extension.pybindings import portable_lib as exec_lib
```

Comment:
This will introduce issue previously -- we have to import exec_lib first then sdpa_with_kv_cache

Reply:
We need bump back up, or the imports fail 

We should comment this so people know in the future